mirage/factories/course-stage-completion.js

```diff
@@ -19,6 +19,11 @@ export default Factory.extend({
       status: nextStage ? 'idle' : 'completed',
     });
 
+    courseStageCompletion.repository.update(
+      'completedStageSlugs',
```

Comment:
@libmartinito isn't `repository.completedStageSlugs` a computed property? Not sure how updating that here helps - this seems inaccurate